src/string_util.h

```diff
@@ -0,0 +1,48 @@
+#ifndef BENCHMARK_STRING_UTIL_H_
+#define BENCHMARK_STRING_UTIL_H_
+
+#include <string>
+#include <sstream>
+#include <utility>
+
+namespace benchmark {
+
+void ToExponentAndMantissa(double val, double thresh, int precision,
```

Comment:
which of these actually need to be in the header? i'm guessing not all.

would you mind checking before you merge?